src/shared/crypto/bitcoin/bitcoin.utils.ts

```diff
@@ -64,24 +65,19 @@ export function decodeBitcoinTx(tx: string) {
 export function getAddressFromOutScript(script: Uint8Array, bitcoinNetwork: BtcSignerNetwork) {
   const outputScript = btc.OutScript.decode(script);
 
```

Comment:
How about changing this to use a `switch` statement for readability? 🤔

Reply:
Yeah, prob a good idea, will do. 👍

Reply:
I think that looks much better but yesterday @kyranjamie suggested I don't use `switch` on my PR in favour of early return. 

I think this is a nice situation for `switch` but maybe we should keep consistency with early return. 

Sorry @fbwoolf to ask you to change it then back again 🤦

Reply:
@kyranjamie what is the pref here?